06_deploy_masters.sh

```diff
@@ -18,9 +18,9 @@ wait_for_json ironic \
 nodes=$(openstack baremetal node list)
 for node in $(jq -r .nodes[].name ${instack}); do
   if [[ $nodes =~ $node ]]; then
-    openstack baremetal node undeploy $node;
-    openstack baremetal node delete $node;
-  fi;
+    openstack baremetal node undeploy --wait $node || true
```

Comment:
you need to add the "--wait" as the last param, otherwise it thinks the following param is the amount of seconds to wait for...

Reply:
Oops thanks, now fixed